sigmastate/src/main/scala/sigmastate/eval/Zero.scala

```diff
@@ -19,7 +19,7 @@ case class CZero[T](zero: T) extends Zero[T]
 
 trait ZeroLowPriority {
   implicit def collIsZero[T: Zero: RType]: Zero[Coll[T]] = CZero(Colls.emptyColl[T])
-  implicit def optionIsZero[T: Zero]: Zero[Option[T]] = CZero(Some(Zero.zeroOf[T]))
```

Comment:
Is this change safe? Used in EvalSizeBox.getReg, are implementations getting the same result there for 

```
        val z = Zero.typeToZero(tT)
        val zeroVal = z.zero
        val sized = Sized.typeToSized(tT)
        sized.size(zeroVal)
```

?

Reply:
rolled it back.